resources.whatwg.org/build/deploy.sh

```diff
@@ -78,20 +77,17 @@ run_post_build_step() {
     fi
 }
 
-if [[ $BRANCH != "master" ]] ; then
-    # Branch snapshot, if not master
-    BRANCH_DIR="$WEB_ROOT/$BRANCHES_DIR/$BRANCH"
-    mkdir -p "$BRANCH_DIR"
-    curl https://api.csswg.org/bikeshed/ -f -F file=@"$INPUT_FILE" -F md-status=LS-BRANCH \
-         -F md-warning="Branch $BRANCH $BRANCH_URL_BASE$BRANCH replaced by $LS_URL" \
-         -F md-title="$TITLE (Branch Snapshot $BRANCH)" \
-         -F md-Text-Macro="SNAPSHOT-LINK $BACK_TO_LS_LINK" \
-         > "$BRANCH_DIR/index.html";
-    copy_extra_files "$BRANCH_DIR"
-    run_post_build_step "$BRANCH_DIR"
-    echo "Branch snapshot output to $BRANCH_DIR"
-else
-    # Commit snapshot, if master
+# Living standard
+curl https://api.csswg.org/bikeshed/ -f -F file=@"$INPUT_FILE" \
+     -F md-Text-Macro="SNAPSHOT-LINK $SNAPSHOT_LINK" \
+     > "$WEB_ROOT/index.html"
+copy_extra_files "$WEB_ROOT"
+run_post_build_step "$WEB_ROOT"
+echo "Living standard output to $WEB_ROOT"
+echo ""
+
+# Commit snapshot
```

Comment:
If the intention is to run this for pull requests too, we should probably omit the commit snapshot, as it doesn't provide extra validation and just makes pull requests take longer. Same with the service worker.

Reply:
I was rather thinking the other way around, build for PRs instead of branches. Let's discuss in the separate PR I'm sending.